pkg/services/sqlstore/migrations/dashboard_mig.go

```diff
@@ -234,4 +234,8 @@ func addDashboardMigration(mg *Migrator) {
 	mg.AddMigration("Add isPublic for dashboard", NewAddColumnMigration(dashboardV2, &Column{
 		Name: "is_public", Type: DB_Bool, Nullable: false, Default: "0",
 	}))
+
+	mg.AddMigration("Add deleted for dashboard", NewAddColumnMigration(dashboardV2, &Column{
+		Name: "deleted", Type: DB_DateTime, Nullable: true,
```

Comment:
❤️  (i came here to check if it was a time or a boolean... and am happy with what I found!)